packages/unified-pipeline/src/components/Canvas/elements/Nodes/AtomicNode/AtomicNode.tsx

```diff
@@ -105,11 +105,11 @@ export default function AtomicNode({ isConnectable, data, id, xPos, yPos, zIndex
           <div
             style={{
               width,
-              height
+              minHeight: height
```

Comment:
added `minHeight` as we need to increase the node height if displayName is big

From the mock:
<img width="828" alt="image" src="https://github.com/user-attachments/assets/16303ebb-60a5-417e-9002-8edbe68fc0fa">

Reply:
This should be ok for now but we might want to wrap long step names to a single line with `...`

Reply:
we want to support two lines of text here. dynamic height is a good solution, as long as it doesn't break any layout